todo_app/flask_config.py

```diff
@@ -1,9 +1,11 @@
-import os
-
-
-class Config:
-    def __init__(self):
-        """Base configuration variables."""
-        self.SECRET_KEY = os.environ.get('SECRET_KEY')
-        if not self.SECRET_KEY:
-            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+import os
+from dotenv import load_dotenv
+
+
+class Config:
+    def __init__(self):
+        """Base configuration variables."""
+
+        self.SECRET_KEY = os.environ.get('SECRET_KEY')
+        if not self.SECRET_KEY:
+            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")

```

Comment:
You've got lots of random file changes that aren't obvious what actually changes - I'm guessing something funny has happened with line endings!